File: core/src/server/model/ai/strategy/AttackStrategy/AttackStrategy.scala
package server.model.ai.strategy.AttackStrategy

import server.model.ai.EasyComputerPlayer
import server.model.playerData.Region
import java.util.HashSet

abstract trait AttackStrategy {
    def attackTargets(p: EasyComputerPlayer, myTargets: HashSet[Region])
}

File: core/src/server/model/ai/strategy/AttackStrategy/AttackStrategy.java
package server.model.ai.strategy.AttackStrategy;

import server.model.ai.EasyComputerPlayer;
import server.model.playerData.Region;
import java.util.HashSet;

public interface AttackStrategy {
    public void attackTargets(EasyComputerPlayer p,HashSet<Region> myTargets);
}
